sql/mob_pools.sql

```diff
@@ -6067,7 +6067,7 @@ INSERT INTO `mob_pools` VALUES (6007,'aldo_uc','Aldo',149,0x0000F90B000000000000
 INSERT INTO `mob_pools` VALUES (6008,'naja_uc','NajaSalaheem',151,0x0000FA0B00000000000000000000000000000000,6,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,417,0,0,1123,151);
 INSERT INTO `mob_pools` VALUES (6009,'lion_ii','Lion',149,0x0000090C00000000000000000000000000000000,6,13,3,240,100,0,0,0,0,0,0,32,0,3,0,0,418,0,0,1124,149);
 INSERT INTO `mob_pools` VALUES (6010,'zeid_ii','Zeid',146,0x00000E0C00000000000000000000000000000000,8,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,419,0,0,1125,146);
-INSERT INTO `mob_pools` VALUES (6011,'prishe_ii','Prishe',145,0x00000A0C00000000000000000000000000000000,2,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,420,0,0,1126,145);
+INSERT INTO `mob_pools` VALUES (6011,'prishe_ii','Prishe',145,0x00000A0C00000000000000000000000000000000,3,2,3,240,100,0,0,0,0,0,0,32,0,3,0,0,420,0,0,1126,145);
```

Comment:
So after doing Monstrosity, I do wonder if Trusts should be treated as equal level main job and sub job, if this is to line up their traits and stats etc.

Reply:
I could see that being the case, can potentially help research that later on

Reply:
@CatsEyeXI are you setting her whm/mnk instead of mnk/whm? 

@zach2good the way we handle mobs traits in general isn't quite right, but they do get from both jobs at lv of main. They aren't like players where they always get the 2nd job halved - they just don't get full stats of their 2nd job instead. Certain standard job traits aren't on all mobs of that job though and at the time we decided to make jobs auto-add their traits we didn't knw this. To get it to 100% retail accurate we'll need the ability to just bolt on or "equip" if you will, traits. Just like how we can arbitrarily add spells. I ***do not*** want people compensating for missing traits by just using a bunch of `addMod` in scripts or annoying sql modifier rows because in the past that lead to people unknowingly stacking multiple sources of the same bonus. That way leads to unsleepable paladins and warriors with for more double attack than ever intended..

Reply:
iirc Frank already set our monsters to have full sj levels and modified stats. Trusts likely need an adjustment to match.

Reply:
> @CatsEyeXI are you setting her whm/mnk instead of mnk/whm?
> 
> @zach2good the way we handle mobs traits in general isn't quite right, but they do get from both jobs at lv of main. They aren't like players where they always get the 2nd job halved - they just don't get full stats of their 2nd job instead. Certain standard job traits aren't on all mobs of that job though and at the time we decided to make jobs auto-add their traits we didn't knw this. To get it to 100% retail accurate we'll need the ability to just bolt on or "equip" if you will, traits. Just like how we can arbitrarily add spells. I _**do not**_ want people compensating for missing traits by just using a bunch of `addMod` in scripts or annoying sql modifier rows because in the past that lead to people unknowingly stacking multiple sources of the same bonus. That way leads to unsleepable paladins and warriors with for more double attack than ever intended..

There's conflicting information on this, But I do believe her to be a WHM/MNK based on the fact she must have a sufficiently large MP pool for Curaga without any mods.

BGWiki and FFXIClopedia do not agree on this.
![image](https://github.com/LandSandBoat/server/assets/65316697/69af312b-ad38-49be-b92f-c1d7cc22669a)
![image](https://github.com/LandSandBoat/server/assets/65316697/65d0f598-9c13-4297-a503-132ca8b4bd0d)

Reply:
the MP issue tells me we're still stating trusts like players and not mobs. even a war/blm on retail will have a ton of mp, because they aren't having their subjobs mp pool nerfed to hell like us.